deprecated/classes/Pods.php

```diff
@@ -500,8 +500,8 @@ public function get_pod_id() {
 	/**
 	 * Search and filter records
 	 *
-	 * @since      1.x
-	 * @deprecated deprecated since version 2.0
+	 * @since 1.x.x
```

Comment:
I assume the '1.x.x' entries are details lost in the depths of time.  Are these Kosher to leave like this?

Reply:
I found and added in a couple of missing values for `@deprecated`, but wanted to get this PR in for 2.7.4.